### ISSUE TITLE
Physical track tables

### ISSUE THREAD

**Issue by andkov** [IALSA/IALSA-2015-Portland#155]

**Comment by ampiccinin** [comment]
thanks! Can you please check the axes? I think some of the labels may be misplaced? Some odd things like grip where I think it should be PEF (based on the magnitude)?

**Comment by andkov** [comment]
@ampiccinin, yes, I have examined the `.out` files and corrected the model names (https://github.com/IALSA/IALSA-2015-Portland/commit/97f33af91a82b122f2e839413a1f4688e7ba059c) the next iteration of reports should sort these two models out and draw the correct graphs and tables

**Comment by andkov** [comment]
@ampiccinin , the new report with corrected names for EAS has been upload and is available after the link above

**Comment by ampiccinin** [comment]
N for HRS too low. Still pointing to old files? Thanks!

**Comment by andkov** [comment]
No, all the new files have been added and processed. But there were only 12 models. The others are old and have low N. I'll double check to make sure, though.

**Comment by andkov** [comment]
You can see this from the fact, that not all HRS N are low.

**Comment by andkov** [comment]
Please let me know if you catch a specific example of where it should be higher than it is.

**Comment by andkov** [comment]
@ampiccinin , I think i found the point of confusion. The files that Chenkai sent the latest, the ones  you pointed out having short names and high Ns (I've stored them separately here: [`./studies/hrs/physical-cognitive/from-email-2016-10-31)`](https://github.com/IALSA/IALSA-2015-Portland/tree/master/studies/hrs/physical-cognitive/from-email-2016-10-31)) are all physical-cognitive. Naturally, this tables of phys-phys do not reflect them. The phys-phys models, while containing the estimated correlations have only ~280 observation ([example](https://github.com/IALSA/IALSA-2015-Portland/blob/master/studies/hrs/physical/b1_female_aehplus_grip_gait.out#L95))

Are there phys-phys models that I've neglected to include? Please let me know, if you recover any.
